GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -33,6 +33,7 @@ public CommitInfo()
             {
                 _sortedRefs = null;
             };
+            _RevisionHeader.Font = new System.Drawing.Font(System.Drawing.FontFamily.GenericMonospace, AppSettings.Font.Size);
```

Comment:
If the user has already chosen a monospaced font, it should not be changed to some other monospaced one.